app/Http/Controllers/Admin/PetShop/OwnerCrudController.php

```diff
@@ -99,4 +100,52 @@ protected function setupUpdateOperation()
     {
         $this->setupCreateOperation();
     }
+
+    protected function setupShowOperation()
+    {
+        $this->setupListOperation();
+
+        Widget::add([
+            'type'       => 'datatable',
+            'controller' => 'App\Http\Controllers\Admin\PetShop\PetCrudController',
+            'name'       => 'pets_crud',
+            'section'    => 'after_content',
+            'wrapper'    => ['class' => 'mt-3'],
+            'content'    => [
+                'header' => 'Pets for this owner',
+                // COULD-DO: maybe add support for a subheader?
+                // 'subheader' => 'This is a list of all pets owned by this owner.',
+            ],
+            // MUST-DO: How the fuck do I make this only show related pets?!?!
+            'configure' => function ($crud, $parent) {
+                // only show the pets of this owner (owner is an n-n relationship)
+                if ($parent) {
+                    $crud->addClause('whereHas', 'owners', function ($query) use ($parent) {
+                        $query->where('id', $parent->id);
+                    });
+                }
+            },
+            // SHOULD-DO: how do I make a new entry automatically related to the owner?
+        ]);
+
+        \Log::info($this->crud->settings());
+
+        Widget::add([
+            'type'       => 'datatable',
+            'controller' => 'App\Http\Controllers\Admin\PetShop\InvoiceCrudController',
+            'name'       => 'invoices_crud',
+            'section'    => 'after_content',
+            'wrapper'    => ['class' => 'mt-3'],
+            'content'    => [
+                'header' => 'Invoices for this owner',
+            ],
+            // MUST-DO: How the fuck do I make this only show related pets?!?!
+            'configure' => function ($crud, $parent) {
+                // only show the pets of this owner (owner is an n-n relationship)
```

Comment:
Trying to change a column label does not work (but other things like removing columns and export buttons DO work):

```
                // why can't I manipulate the passport.number column here?
                $crud->column('skills')->label('Skizzo skills');
```